src/pages/index.js

```diff
@@ -100,14 +182,20 @@ function handleEditButton() {
 }
 
 function handleAddFormSubmit(inputs) {
-  const { title: name, link } = inputs;
-  const card = {
-    name: name,
-    link: link,
-  };
-  const cardElement = renderCard(card);
-  cardsList.prependCard(cardElement);
-  popupCardForm.close();
+  api
+    .addCard(inputs)
+    .then((res) => {
+      const cardElement = renderCard(res);
+      cardsList.prependCard(cardElement);
+      popupCardForm.close();
+      cardForm.reset();
```

Comment:
According to the task you need to have method `close` inside `PopupWithForm` where you need to **reset** the form not to duplicate it in `index.js`